### PR TITLE
HHVM 4.153

### PR DESCRIPTION
**Pull request by fredemmott**
- Announce 4.153.0
- work through commit messages from 3/1 to 3/16

### CODE REVIEW

File: _posts/2022-03-16-hhvm-4.153.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: "HHVM 4.153"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.153 is released! This release marks the end of support of 4.146; 4.147 -
+4.151 remain supported, as do the 4.102 and 4.128 LTS releases. HHVM 4.152 was
```
Comment (Atry) [inline _posts/2022-03-16-hhvm-4.153.markdown:9]: Shall we mention the 4.102.7 version we published this week?

Comment (fredemmott) [reply _posts/2022-03-16-hhvm-4.153.markdown:9]: oh, also need the LTS + 2 week cycle stuff doing from my comments on previous PR

**Review by Atry — APPROVED** [review]

File: _posts/2022-03-16-hhvm-4.153.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: "HHVM 4.153"
+layout: post
+author: atry
```
Comment (Atry) [inline _posts/2022-03-16-hhvm-4.153.markdown:4]:
```suggestion
author: fredemmott
```

**Comment by AndrewDiMola** [comment]
re: inout parameters can now be dynamic.

We should probably update this section: https://docs.hhvm.com/hack/functions/inout-parameters#dynamic-usage

> `inout` is a different calling convention, so dynamic calls will not work with `inout` parameters. For example, you cannot use [meth_caller](https://docs.hhvm.com/hack/reference/function/HH.meth_caller/), `call_user_func` or [ReflectionFunction::invoke](https://docs.hhvm.com/hack/reference/class/ReflectionFunction/invoke/).

Is this now all allowable?

---

Same for https://docs.hhvm.com/hack/built-in-types/resources, but might be fine since it doesn't even mention dynamic, unless it now should.

---

And on https://docs.hhvm.com/hack/built-in-types/dynamic: neither mention of resource or inout, but maybe that's totally fine too.

**Comment by fredemmott** [comment]
> We should probably update this section: https://docs.hhvm.com/hack/functions/inout-parameters#dynamic-usage

'Dynamic calls' means things like `$foo();` and are unchanged; this is distinct to https://docs.hhvm.com/hack/built-in-types/dynamic - which is what's being referred to here

dynamic calls might or might not have `dynamic` parameters, and functions with `dynamic` parameters can be called in non-dynamic ways

**Comment by fredemmott** [comment]
Given timing and that the builds are already in our repositories, I've pushed this - I'll be keeping an eye out for any post-commit feedback.

**Pull request merged**